persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/field/MongoMapField.scala

```diff
@@ -91,7 +91,7 @@ class MongoMapField[OwnerType <: BsonRecord[OwnerType], MapValueType](rec: Owner
   def asDBObject: DBObject = {
     val dbo = new BasicDBObject
     value.keys.foreach { k =>
-      dbo.put(k.toString, value.getOrElse(k, ""))
+      value.get(k).foreach(v => dbo.put(k.toString, v.asInstanceOf[Object]))
```

Comment:
There are enough single letter variable names here to be confusing. Can we do something like...

```scala
value.keys.foreach { key =>
  value.get(key).foreach { innerValue =>
    dbo.put(key.toString, innerValue.asInstanceOf[Object])
  }
}
```

?